tests/standalone/SixUsing.py

```diff
@@ -0,0 +1,41 @@
+import six
+import sys
+import unittest
+
+
+def dispatch_types(value):
+   
+    if isinstance(value, six.integer_types):
+        return int
+
+    elif isinstance(value, six.string_types):
+        return str
+    
+    elif isinstance(value, six.binary_type):
+        return bytes
+
+def get_iterator(dictionary):
+    return six.viewkeys(dictionary), six.viewvalues(dictionary), six.viewitems(dictionary)
+
+
+class TestAssertCountEqual(unittest.TestCase):
+    def test(self):
+        six.assertCountEqual(self, (1, 2), [2, 1])
+        
+
+def main():
+    print(dispatch_types(6))
+    print(dispatch_types('hi'))
+    u='hi'
+    print(dispatch_types(u.encode('utf-8')))
+    Dict = {'Tim': 18,'Charlie':12,'Tiffany':22,'Robert':25} 
```

Comment:
Just a minor quibble, don't name local variables this way, makes me think it's a class or when I see it used.

As for the prints, import the `print_function` from future, and add some identifying prefixes, e.g. `dispatch_types int:`, the so output is more human readable. It works the way you did it, but it's not nice.